src/toil/test/lib/dockerTest.py

```diff
@@ -106,13 +113,13 @@ def testDockerPipeChain(self, caching=True):
         printf 'x\n y\n' | wc -l
         """
         options = Job.Runner.getDefaultOptions(os.path.join(self.tempDir, 'jobstore'))
-        options.logLevel = 'INFO'
+        options.logLevel = 'OFF'
         options.workDir = self.tempDir
         options.clean = 'always'
-        if not caching:
-            options.disableCaching = True
+        options.disableCaching = True
```

Comment:
Why are we now ignoring the caching option?

Reply:
Thanks, I missed that.  Definitely should not have been changed.